web/html/xui/xray.html

```diff
@@ -82,11 +84,11 @@
                                     <a-button type="danger" :disabled="!saveBtnDisable" @click="restartXray">{{ i18n "pages.xray.restart" }}</a-button>
                                     <a-popover v-if="restartResult"
                                         :overlay-class-name="themeSwitcher.currentTheme">
-                                        <span slot="title" style="font-size: 12pt">Error in running xray-core</span>
+                                        <span slot="title" style="font-size: 12pt">An error occurred while running Xray</span>
```

Comment:
Not a good replacement.
Errors that we use here will stop xray-core. `while` is not correct word.